pkg/util/chunk/chunk_in_disk.go

```diff
@@ -0,0 +1,316 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package chunk
+
+import (
+	"io"
+	"os"
+	"strconv"
+	"unsafe"
+
+	errors2 "github.com/pingcap/errors"
+	"github.com/pingcap/tidb/pkg/parser/terror"
+	"github.com/pingcap/tidb/pkg/types"
+	"github.com/pingcap/tidb/pkg/util/disk"
+	"github.com/pingcap/tidb/pkg/util/memory"
+)
+
+const byteLen = int64(unsafe.Sizeof(byte(0)))
+const intLen = int64(unsafe.Sizeof(int(0)))
+const int64Len = int64(unsafe.Sizeof(int64(0)))
+
+const chkFixedSize = intLen + 3*intLen
```

Comment:
how about: or add comment why split them?
```suggestion
const chkFixedSize = 4*intLen
```

Reply:
> how about: or add comment why split them?

okk, I will merge them.